examples/material-ui-nextjs/src/components/ThemeRegistry/EmotionCache.js

```diff
@@ -1,5 +1,4 @@
 'use client';
-
 import * as React from 'react';
 import createCache from '@emotion/cache';
```

Comment:
To match the rest of the codebase, but React documents this with a space https://react.dev/reference/react/use-client#use-client. Should we refactor all our codebase @mj12albert?